apps/frontend/src/pages/quiz/libs/components/balance-wheel/balance-wheel.tsx

```diff
@@ -40,6 +41,23 @@ const BalanceWheel: React.FC = () => {
 		});
 	}, []);
 
+	const handleNavigate = useCallback(() => {
+		const isDone =
+			percentage >= PercentageConfig.MAX_VALUE &&
+			hasAnsweredQuizQuestions &&
+			hasAnsweredOnboardingQuestions;
```

Comment:
The QA notice that the user is redirected while the percentage is still displayed at 88%. So I added a delay to the redirect in order for the "Analyzing 100%" to be displayed and redirect the user.

Reply:
The current implementation redirects the user to root page when the percentage is at 100% but the user cannot see the 100% because it quickly redirect to the root page.